Kickstarter-iOS/AppDelegate.swift

```diff
@@ -238,11 +239,9 @@ internal final class AppDelegate: UIResponder, UIApplicationDelegate {
     open url: URL,
     options: [UIApplication.OpenURLOptionsKey: Any] = [:]
   ) -> Bool {
-    return self.viewModel.inputs.applicationOpenUrl(
-      application: app,
-      url: url,
-      options: options
-    )
+    self.viewModel.inputs.applicationOpenUrl(application: app, url: url, options: options)
+
+    return self.fbLoginViewModel.inputs.applicationOpenUrl(application: app, url: url, options: options)
```

Comment:
This pings our `AppDelegateViewModel` first so that deep-links are handled and then returns the value from `FBSDKApplicationDelegate`. I considered doing something like:

```swift
let deepLinkHandled = self.viewModel.inputs.applicationOpenUrl(application: app, url: url, options: options)

guard deepLinkHandled else {
  return self.fbLoginViewModel.inputs.applicationOpenUrl(application: app, url: url, options: options)
}

return deepLinkHandled
```
But in fact leaving it the way it is now more closely resembles how it's been all along and I'm just not sure if this is something I want to change at this stage 🤔. Open to thoughts on that.